### ISSUE TITLE
Accessibility enhancements

### ISSUE THREAD

**Issue by seanohue** [shawncplus/ranviermud#170]
This might require some refactoring, but I think that it would be nice to include a couple of options to keep the framework accessible from the start:

1) Colorblind: I don't _think_ there are many things that are denoted solely by color but if there are it'd be nice to enable more verbose labels, or to configure the color scheme considering that a lot of messaging relies on red and green.

2) Visually-impaired: Most of the game uses text to get the point across, the one exception I can think of are the health bars during combat. There should be a toggle to not show the health bars at all, for those who may be using screen readers.

**Comment by seanohue** [comment]
3. Again, for those using screen readers, the ability to configure the prompt, including only showing the player character's status on command.

**Comment by seanohue** [comment]
4. Related to point 2 above, adding an option to show health percentage or raw numbers rather than the bars during combat.

**Comment by shawncplus** [comment]
This is a bundle feature, not a core feature. I don't know why I labelled it as such. The health bars come from the `ranvier-combat` bundle which I'm fine with leaving as-is, if someone wants to change that it's just a matter of removing the `addPrompt` lines in the `player-events.js` in that bundle.
